include/vcpkg/cmake.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <vcpkg/base/stringview.h>
+
+namespace vcpkg
+{
+    StringView find_cmake_invocation(StringView content, StringView command);
```

Comment:
Given that these functions are _very much_ heuristics and only usable with caveats, I don't think they should be introduced as an independent "cmake" module. Instead, I think they should remain in spdx.cpp and exposed as needed for testing purposes.

Ideally, we will get rid of these heuristics entirely and instead introduce some sort of "side channel" that allows the portfile to record accesses -- such as by writing out a json document inside `vcpkg_from_github()` with the actual information.

If we do introduce another area that needs CMake parsing heuristics before we're able to delete them from the SPDX path, we can move them out into separate files at that point.